<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 rgba(248, 250, 228, 255),
                                stop:0.5 rgba(247, 249, 252, 255));
    border: 1px solid gray;
}
QWidget {
    border: none;
}

QPushButton {
    border: none;
}

QLabel {
    border: none;
}

QLineEdit {
    border: none;
}

QTextEdit {
    border: none;
}

QComboBox {
    border: none;
}

QSpinBox {
    border: none;
}

QDoubleSpinBox {
    border: none;
}

QSlider {
    border: none;
}

QScrollBar {
    border: none;
}

/* 其他部件... */</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">centralwidget {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 rgba(248, 250, 228, 255),
                                stop:0.5 rgba(247, 249, 252, 255));
    border: 1px solid gray;
}</string>
   </property>
   <widget class="QSlider" name="m_volumeSlider">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>470</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QStackedWidget" name="fatherStackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1061</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_4">
     <property name="styleSheet">
      <string notr="true">KxMusicListWidget{
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 rgba(248, 250, 228, 255),
                                stop:0.5 rgba(247, 249, 252, 255));
}</string>
     </property>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>111</width>
        <height>551</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 rgba(248, 250, 228, 255),
                                stop:0.5 rgba(247, 249, 252, 255));
    border: 1px solid gray;
	bottom-border: none
}
QListWidget {
    color: black;
}

QListWidget::item {
    padding-top: 10px;
    padding-bottom: 10px;
    margin: 5px;
}

QListWidget::item:selected {
    background-color: lightblue;
    color: black;
}</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>911</width>
        <height>541</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">stackedWidget {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 rgba(248, 250, 228, 255),
                                stop:0.5 rgba(247, 249, 252, 255));
    border: 1px solid gray;
}
KxMusicList::item {
    /* 设置每个 QListWidgetItem 的高度 */
    height: 50px; /* 更改此值以调整项目的高度 */
    /* 设置项目的边距 */
    margin: 5px;
    /* 设置项目的内边距 */
    padding: 5px;
    /* 设置项目的字体 */
    font-family: &quot;宋体&quot;;
    font-size: 12pt;
    font-weight: bold;
    color: black;
}

</string>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="KxMusicListWidget" name="m_currentMusicList">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>841</width>
          <height>531</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">m_currentMusicList{
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 rgba(248, 250, 228, 255),
                                stop:0.5 rgba(247, 249, 252, 255));
    border: 1px solid gray;
}</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="KxMusicListWidget" name="m_internetMusicList">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>841</width>
          <height>531</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="KxMusicListWidget" name="m_localMusicList">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>841</width>
          <height>531</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="KxRotatableLabel" name="RoLabel">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="musicNameLabel">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>10</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="albumNameLabel">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>40</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextEdit" name="m_textEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>331</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="m_lyricWidget" native="true">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>90</y>
        <width>481</width>
        <height>421</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    font-family: &quot;宋体&quot;;
    font-size: 12pt;
    font-weight: bold;
    color: black;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_1">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">QLabel {
    font-family: &quot;宋体&quot;;
    font-size: 18pt;
    font-weight: bold;
    color: black;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>550</y>
      <width>1151</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QWidget {
    border: none;
	border: 1px solid gray;
 background-color: rgb(250, 250, 250);
}

QPushButton {
    border: none;
}

QLabel {
    border: none;
}

QLineEdit {
    border: none;
}

QTextEdit {
    border: none;
}

QComboBox {
    border: none;
}

QSpinBox {
    border: none;
}

QDoubleSpinBox {
    border: none;
}

QSlider {
    border: none;
}

QScrollBar {
    border: none;
}

/* 其他部件... */</string>
    </property>
    <widget class="QPushButton" name="m_buttonPlayer">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>15</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="m_addButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>20</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="m_requestButton">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>20</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSlider" name="m_positionSlider">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>90</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="m_positionLabel">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>90</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>00:00</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="m_infoLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextEdit {
    font-family: &quot;宋体&quot;;
    font-size: 9pt;
    font-weight: bold;
    color: black;
}</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonNext">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>15</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="m_mainLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPre">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>15</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="m_imageButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KxMusicListWidget</class>
   <extends>QListWidget</extends>
   <header>kxmusiclistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KxRotatableLabel</class>
   <extends>QLabel</extends>
   <header>kxrotatablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
